website/docs/cli/commands/describe/describe-affected.mdx

```diff
@@ -713,3 +715,72 @@ private repo which is not itself):
     ```shell
     atmos describe affected --repo-path $GITHUB_WORKSPACE
     ```
+
+## Upload the affected components and stacks to an HTTP endpoint
+
+If the `--upload=true` command-line flag is passed, Atmos will upload the affected components and stacks to a
+specified HTTP endpoint.
+
+The endpoint can process the affected components and their dependencies in a CI/CD pipeline (e.g. execute
+`terraform apply` on all the affected components in the stacks and all the dependencies).
+
+Atmos will perform an HTTP POST request to the URL `${ATMOS_PRO_BASE_URL}/${ATMOS_PRO_ENDPOINT}`, where the base URL
+is defined by the `ATMOS_PRO_BASE_URL` environment variable, and the URL path is defined by the `ATMOS_PRO_ENDPOINT`
+environment variable.
+
+An [Authorization](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization) header
+`Authorization: Bearer $ATMOS_PRO_TOKEN` will be added to the HTTP request (if the `ATMOS_PRO_TOKEN` environment
+variable is set) to provide credentials to authenticate with the server.
+
+:::note
+If the `--upload=true` command-line flag is passed, the `--include-dependencies` flag is automatically set to `true`,
+so the affected components will always be uploaded with their dependencies (if they are configured in Atmos stack manifests).
+:::
+
+The payload of the HTTP POST request will be a JSON object with the following schema:
+
+<Terminal title="Payload of 'atmos describe affected --upload=true' command">
+  ```json
+    {
+       "base_sha": "6746ba4df9e87690c33297fe740011e5ccefc1f9",
+       "head_sha": "5360d911d9bac669095eee1ca1888c3ef5291084",
+       "repo_url": "https://github.com/cloudposse/atmos",
+       "stacks": [
+          {
+            "component": "vpc",
+            "component_type": "terraform",
+            "component_path": "examples/quick-start/components/terraform/vpc",
+            "stack": "plat-ue2-dev",
+            "stack_slug": "plat-ue2-dev-vpc",
+            "affected": "stack.vars",
+            "included_in_dependents": false,
+            "dependents": []
```

Comment:
I mentioned it in slack, but in `v1.81.0`,  atmos seems to be dropping `dependents` if it's empty. We need to make sure it's always included, even when empty.